pytket/phir/phirgen.py

```diff
@@ -278,35 +313,7 @@ def genphir(
 
     # TODO(kartik): this may not always be accurate
     # https://github.com/CQCL/pytket-phir/issues/24
```

Comment:
this comment is for the `qvar_dim` and `cvar_dim` calculations and hence should be moved inside the function above.